agent/06_agent_create_cluster.sh

```diff
@@ -112,6 +112,38 @@ function attach_agent_iso() {
 
 }
 
+function attach_appliance_diskimage() {
```

Comment:
It looks like this function populates the $OCP_DIR with several artifacts, but the `make clean` is broken as the `cache` and `temp` folder are created under the `root` user. Please add the required cleaning steps so that it would be possible to remove everything when running the `clean` target